### PR TITLE
ReadMaterials Process uses Kratos::Parameters now instead of json

### PR DESCRIPTION
**Pull request by philbucher**
See title, tests are also working

### CODE REVIEW

**Review by loumalouomega — APPROVED** [review]
Looks nice

File: kratos/python_scripts/read_materials_process.py [hunk 1]
```diff
@@ -111,40 +108,32 @@ def _AssignPropertyBlock(self, data):
...
-                else:
-                    vector = Vector(size_1)
-                    for i in range(size_1):
-                        vector[i] = value[i]
-                    prop.SetValue( var, vector)
+            if value.IsMatrix():
+                prop.SetValue( var, value.GetMatrix() )
+            elif value.IsVector():
+                prop.SetValue( var, value.GetVector() )
             else:
```
Comment (marandra) [inline kratos/python_scripts/read_materials_process.py]: I think it is missing the cases when value is of type String and Boolean

Comment (philbucher) [reply kratos/python_scripts/read_materials_process.py]: Good point!
I changed it accordingly, now you can use all variable types that are available in Kratos::Parameters

**Comment by philbucher** [comment]
@RiccardoRossi any requests for changes?
Otherwise I merge beginning of next week

**Pull request merged**